source/backend/apimodels/Models/Concepts/AcquisitionFile/AcquisitionFilePropertyMap.cs

```diff
@@ -30,6 +30,16 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.DisplayOrder, src => src.DisplayOrder)
                 .Map(dest => dest.Location, src => src.Location)
                 .Inherits<BaseConcurrentModel, Entity.IBaseEntity>();
+
+
+            config.NewConfig<Entity.PimsPropertyAcquisitionFileHist, Entity.PimsPropertyAcquisitionFile>()
```

Comment:
something wacky going on with the formatting here. Are you manually formatting files?

Reply:
yeh, weird. I will run the formatter directly and see if that helps it.